tests/foreman/ui/test_hostunification.py

```diff
@@ -129,12 +188,31 @@ def test_positive_rename_foreman_host(self):
 
         @assert: Host appears in both places despite being renamed
 
-        @caseautomation: notautomated
-
         @CaseLevel: System
         """
+        with VirtualMachine(distro='rhel71') as vm:
+            vm.install_katello_ca()
+            result = vm.register_contenthost(self.org_.label, lce='Library')
+            self.assertEqual(result.return_code, 0)
+            with Session(self.browser) as session:
+                session.nav.go_to_select_org(self.org_.name)
+                name, domain_name = vm.hostname.split('.', 1)
+                new_name = gen_string('alphanumeric').lower()
+                self.hosts.update(
+                    name,
+                    domain_name,
+                    new_name=new_name,
+                )
+                # Host rename operation is not atomic and may take some time
+                for _ in range(3):
```

Comment:
Not sure that I get when second iteration of that loop can happen
if searched not None -> leave loop
if None -> fail the test